src/dotnet/APIView/APIView/APIView.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
While I think this is a good change was there a reason you needed it for this change, or what else prompted this change?

Reply:
I needed to use .Net version >=5 to use latest Http classes to handle status code easily. .NetCore 3.1 Http exception doesn't have status code.

Reply:
Cool. .NET 3.1 is going out of support later this month as well so it is a good idea to move anyway.